### PR TITLE
fix: Reports not working on installer 

### PR DESCRIPTION
**Pull request by soggyomelette**
# Description

Fixes the reports not working on installer bug by replacing papermill with nbconvert to convert ipynb to py files on the fly and executes .py instead of .ipynb

### CODE REVIEW

**Review by piiq — CHANGES_REQUESTED** [review]
We should definitely not discard notebooks but rather figure out a way to fix/overcome the underlying issue

**Comment by Chavithra** [comment]
> We should definitely not discard notebooks but rather figure out a way to fix/overcome the underlying issue

We can use that as a patch and fixing that on the next increment.

**Comment by soggyomelette** [comment]
I've been making good progress with parametrising without papermill and using just nbconvert, should be able to tell more in a few hours

**Comment by soggyomelette** [comment]
> We should definitely not discard notebooks but rather figure out a way to fix/overcome the underlying issue

@piiq the new changes should allow us to retain the notebooks without depending on papermill and also allowing parametrisation

File: openbb_terminal/reports/reports_controller.py [hunk 1]
```diff
@@ -209,27 +213,66 @@ def switch(self, an_input: str):
...
+                    )
+
+                notebook_file_copy = notebook_file + "copy"
 
-            if not result["metadata"]["papermill"]["exception"]:
+                with open(notebook_file_copy + ".ipynb", "w") as file:
+                    file.write(notebook_content)
+
+            # use nbconvert to switch to .py
+            os.system(f"jupyter nbconvert --to script --no-prompt {notebook_file_copy}")
```
Comment (piiq) [inline openbb_terminal/reports/reports_controller.py:246]: can you use the nbconvert python api insteas of an os.system call?
see https://nbconvert.readthedocs.io/en/latest/nbconvert_library.html

**Review by piiq — CHANGES_REQUESTED** [review]

File: openbb_terminal/reports/reports_controller.py (hunk 1 above)
Comment (soggyomelette) [reply openbb_terminal/reports/reports_controller.py:246]: Thanks for this, I wasn't able to figure this out from their github

Comment (piiq) [reply openbb_terminal/reports/reports_controller.py:246]: you can also look into nbconverts tooling to load and parse the notebook and get the cells

**Comment by piiq** [comment]
handled here https://github.com/OpenBB-finance/OpenBBTerminal/pull/2570

**Pull request closed without merging**